colophon.php

```diff
@@ -20,7 +20,7 @@
 	 *
 	 * @return void
 	 */
-	function team51_credits( $args = array() ) {
+	function team51_credits( $args = array() ): void {
```

Comment:
It should be fine with the return type declaration here but just acknowledging that mandates php7.

Which I don't think should be a problem, my brain's just always touchy on making sure such things are explicit, having survived the PHP 5.2 and 5.3 support wars back in the day.